src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -249,20 +252,22 @@ public synchronized void applyDDL(Sequenced<DDLEvent> sequencedEvent) throws Exc
         TableId tableId = TableId.of(project, event.getDatabase(), event.getTable());
         Table table = bigQuery.getTable(tableId);
         // TODO: check schema of table if it exists already
-        // TODO: add a way to get PK of a table through Delta API?
         if (table == null) {
           TableDefinition tableDefinition = StandardTableDefinition.newBuilder()
             .setSchema(Schemas.convert(addSequenceNumber(event.getSchema())))
             .build();
-          String description;
-          if (event.getPrimaryKey().isEmpty()) {
-            description = "Primary Key: _sequence_num";
-          } else {
-            description = "Primary Key: " + event.getPrimaryKey().stream().collect(Collectors.joining(","));
+          List<String> primaryKeys = event.getPrimaryKey();
+          if (primaryKeys.isEmpty()) {
+            throw new DeltaFailureException(
+              String.format("Primary key should not be empty for table '%s' in dataset '%s'",
```

Comment:
nit: Table '%s' in Database '%s' has no primary key. Tables without a primary key are not supported.

Reply:
Sure.